internal/service/project/data_source_project.go

```diff
@@ -137,12 +138,40 @@ func (d *projectDS) Schema(ctx context.Context, req datasource.SchemaRequest, re
 					},
 				},
 			},
+			"ip_addresses": schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"services": schema.SingleNestedAttribute{
+						Attributes: map[string]schema.Attribute{
+							"clusters": schema.ListNestedAttribute{
+								NestedObject: schema.NestedAttributeObject{
+									Attributes: map[string]schema.Attribute{
+										"cluster_name": schema.StringAttribute{
+											Computed: true,
+										},
+										"inbound": schema.ListAttribute{
+											ElementType: types.StringType,
+											Computed:    true,
+										},
+										"outbound": schema.ListAttribute{
+											ElementType: types.StringType,
+											Computed:    true,
+										},
+									},
+								},
+								Computed: true,
+							},
+						},
+						Computed: true,
+					},
+				},
+				Computed: true,
```

Comment:
knitpit: i like Computed at the beginning so it's easier to see

Reply:
adjusted